stories/demos/HeaderFilters.tsx

```diff
@@ -23,9 +22,18 @@ const toolbarClassname = css`
 `;
 
 const filterContainerClassname = css`
-  display: flex;
-  height: inherit;
-  align-items: center;
+  .rdg-header-row .rdg-cell:not(:first-child) {
```

Comment:
Alternatively we could set a static class name to filterable columns, then use it here instead of `:not(:first-child)`.